OAuth2/Storage/Moodle.php

```
@@ -293,73 +275,67 @@ public function setRefreshToken($refresh_token, $client_id, $user_id, $expires,
-        if($user){
+        $user = $DB->get_record('user', ['username' => $username]);
+        if ($user) {
             if ($firstName) {
-                $DB->set_field('user','firstname', $firstName, array('id'=>$user->id));
+                $DB->set_field('user', 'firstname', $firstName, ['id' => $user->id]);
             }
             if ($lastName) {
-                $DB->set_field('user','lastname', $lastName, array('id'=>$user->id));
+                $DB->set_field('user', 'lastname', $lastName, ['id' => $user->id]);
             }
             update_internal_user_password($userInfo, $password);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$userInfo'. (undefined)

(UnusedLocalVariable)